pkg/arvo/app/graph-store.hoon

```diff
@@ -41,19 +37,18 @@
   ^-  (quip card _this)
   =+  !<(old=versioned-state old-vase)
   =|  cards=(list card)
-  |^
+  |-
   ?-    -.old
-      %0  
+      %0
+    =*  zro  zero-load:upgrade:store
```

Comment:
Can't we nix state-zero? iirc it was before the breach

Reply:
I'll look at the history and see if that is the case. If so, I'll get rid of it.